vision/cloud-client/detect.py

```diff
@@ -0,0 +1,389 @@
+#!/usr/bin/env python
+
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This application demonstrates how to perform basic operations with the
+Google Cloud Vision API.
+
+For more information, the documentation at
+https://cloud.google.com/vision/docs.
+"""
+
+import argparse
+import io
+import os
+
+from google.cloud import vision
+
+
+def detect_faces(path):
+    """Detects faces in an image."""
+    vision_client = vision.Client()
+
+    with io.open(path, 'rb') as image_file:
+        content = image_file.read()
+
+    image = vision_client.image(content=content)
+
+    faces = image.detect_faces()
+
+    print('Faces:')
+    for face in faces:
+        print ('anger: {anger}\njoy: {joy}\nsurprise: {surprise}\n').format(
+              anger=face.emotions.anger,
+              joy=face.emotions.joy,
+              surprise=face.emotions.surprise)
+    print('')
```

Comment:
Is this strictly necessary or do you just like a blank newline at the end of the program output?